main_test.go

```diff
@@ -22,6 +23,20 @@ func TestMain(m *testing.M) {
 	ctx := context.Background()
 	if wsl.MockAvailable() {
 		ctx = wsl.WithMock(ctx, mock.New())
+
+		// Touch rootfs so that tests work.
+		// For the real tests, .\prepare-repository.ps1 should be ran before running the tests
+		if err := os.MkdirAll(filepath.Dir(rootFs), 0700); err != nil {
```

Comment:
I don’t like that rootFs is inside the repo: running tests in a repo should not change anything in the repo (apart in update golden files mode).

Do you mind changing the logic so that we can have them in a temporary directory for the mock? (same with the emptyRootFs one).
While we are at it, it should probably be `rootFS`, wdyt?

Reply:
I moved it out, at least for the mocked tests. The real ones still use the images directory.

Also renamed to rootFS

Reply:
that’s fine, I can see the use case for the real tests, because you want to keep your rootfses handy :)

Thanks!